install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -498,6 +495,9 @@ bpf:
   # @default -- `[]`
   vlanBypass: ~
 
+# -- Enable BPF clock source probing for more efficient tick retrieval.
+bpfClockProbe: false
```

Comment:
With this change here, the change to `$defaultBpfClockProbe` is no longer needed

Reply:
Uhm, if the other one is left as true, then new deployments will have it enabled, I'm not sure that's what we intend.

Reply:
Really? I was under the impression that the values.yaml values override the defaults.

I'm fine with the `defaultBpfClockProbe` change though. If we want to set a new default for newer versions then we will just add another stanza with an upgradeCompatibility check for the newer version in order to enable it only on new installations and not for existing ones.

Reply:
Ok, Sebastian confirmed what you said, Joe, that values.yaml overrides the configmap, so I've dropped that changed and kept only the values.yaml one.